packages/release-mastermind/src/utils/helper/semantic.ts

```diff
@@ -1,22 +1,22 @@
 export const semantic: string[] = [
-  'bug',
-  'chore',
-  'opt',
-  'optimisation',
-  'style',
-  'maint',
-  'maintenance',
-  'ref',
-  'refactor',
-  'revert',
-  'dep',
-  'deprecated',
-  'removal',
-  'docs',
-  'documentation',
-  'feat',
-  'enhance',
-  'feature',
-  'enhancement',
-  'fix'
-]
+  "bug",
+  "chore",
+  "opt",
+  "optimisation",
+  "style",
+  "maint",
+  "maintenance",
+  "ref",
+  "refactor",
+  "revert",
+  "dep",
+  "deprecated",
+  "removal",
+  "docs",
+  "documentation",
+  "feat",
+  "enhance",
+  "feature",
+  "enhancement",
+  "fix",
```

Comment:
Codacy has a fix for the issue: [Unexpected trailing comma.](https://app.codacy.com/gh/Videndum/action-masterminds/pullRequest?prid=7872005)
```suggestion
  "fix"
```